src/main/config/config.c

```diff
@@ -503,7 +503,7 @@ static void resetConf(void)
 #ifdef GPS
     // gps/nav stuff
     masterConfig.gpsConfig.provider = GPS_UBLOX;
-    masterConfig.gpsConfig.sbasMode = SBAS_AUTO;
+    masterConfig.gpsConfig.sbasMode = SBAS_NONE;
```

Comment:
I could be wrong here but i think you forgot to change this back to SBAS_AUTO...?